packages/kbn-build/src/production/__fixtures__/packages/foo/src/index.js

```diff
@@ -0,0 +1,5 @@
+import bar from '@elastic/bar'; // eslint-disable-line import/no-unresolved
```

Comment:
__note:__ eslint is complaining just about this single line, so I decided to ignore just this line instead of ignoring `__fixtures__/packages` entirely.